src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/HashBuilder.cs

```diff
@@ -1,22 +1,27 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Security.Cryptography;
 using System.Text;
-using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 
 namespace Uno.UI.SourceGenerators.Helpers
 {
 	internal class HashBuilder
 	{
+		private static uint[] _Lookup32 = Enumerable.Range(0, 256).Select(i =>
+		{
+			string s = i.ToString("x2");
+			return ((uint)s[0]) + ((uint)s[1] << 16);
+		}).ToArray();
+
+		private static readonly unsafe uint* _lookup32UnsafeP = (uint*)GCHandle.Alloc(_Lookup32, GCHandleType.Pinned).AddrOfPinnedObject();
```

Comment:
I'm not sure I understand the point of this. I mean the memory leak isn't an issue given this will live for the whole process duration, but it's still not ideal, and I'm not sure what problem is it trying to solve. If that's just a lookup table, what's the point of pinning it here in the first place? You can just index it directly as an array, no?